RealmSwift/Object.swift

```diff
@@ -421,6 +422,15 @@ public final class DynamicObject: Object {
         }
     }
 
+    public subscript(dynamicMember member: String) -> Any? {
```

Comment:
Are there any tests using this? I didn't spot any, but I could have easily missed them with how much stuff is being added in this PR. Since it's just wrapping the existing functionality it doesn't need much.

Reply:
I have added `testDynamicMemberSubscript` to ObjectTests